docs/guides/cloud/debugging/test-replay.mdx

```diff
@@ -0,0 +1,156 @@
+  alt="Test Replay Demo"
+/>
+
+### Run header
+
+The left side of the header displays the Cypress Cloud test run ID #, with the Git branch name along with the spec, platform and browser information. Hovering on this exposes additional test meta data.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/test-replay-header.jpg"
+  alt="Test Replay header"
+/>
+
+### Command Log
+
+The left side of the screen displays the [command log](/guides/core-concepts/cypress-app#Command-Log) just like in the Cypress app, where you can step through, inspect and [time travel](/guides/core-concepts/cypress-app#Time-traveling) debug your test.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/command-log.png"
+  alt="Test Replay Developer Tools panel"
+  width="60%"
+/>
+
+### Developer Tools
+
+Dive into network-level traffic, console events and inspect the application under test just as you can in the browser.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/dev-tools.png"
+  alt="Test Replay Developer Tools panel"
+  width="60%"
+/>
+
+### Replay Controls
+
+Toggle each test attempt, play back errors at a variety of speeds or scrub through the captured error by dragging the playhead along the timeline.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/scrubber.png"
+  alt="Test Replay scrubber controls"
+/>
+
+## Opt out of Test Replay
+
+:::warning
+
+<Icon name="exclamation-triangle" /> Do not use sensitive data in your tests. This
+will be visible in the test replay.
+
+:::
+
+If you do not want to capture test data for replay and debugging purposes simply "shut off" Test Replay in project settings.
+
+<DocsImage
+  src="/img/guides/cloud/test-replay/opt-out.png"
+  alt="Test Replay opt-out controls"
+/>
+
+## Troubleshooting
+
+**If Test Replay is not present or disabled in Cypress Cloud:**
+
+1. Confirm runs have been recorded to Cypress Cloud using Cypress `v13` or later.
+2. Ensure tests were recorded using a Chromium-based browser(Chrome, Edge, Electron).
+3. Check that Test Replay is enabled in project settings.
+
+**If performance loss is experienced in Cypress with Test Replay enabled:**
+
+Cypress may become less performant under certain circumstances with Test Replay. You may be able to balance performance by taking some additional measures.
+
+Cypress uses a powerful [command log](/guides/core-concepts/cypress-app#Command-Log) locally to display every command executed in your spec. Yet, the command log in the test replay on Cypress Cloud is recreated at a _later_ time. If you are not using screenshots or video or don't need the command log displayed in these artifacts, you can [disable the command log](/guides/references/troubleshooting#Disable-the-Command-Log) when running the test.
+
+This can positively affect test speed without loosing the interactive benefit of the command log during the Test Replay. You can still keep the command log enabled for test runs and artifacts in non-Chromium browsers.
```

Comment:
we made a decision to disable the command log by default - we need to update docs for that and update this section @jaffrepaul cc @jennifer-shehane